pkg/setup/templates/1.10.go

```diff
@@ -21,7 +21,6 @@ ExecStart={{.RootABSPath}}/bin/hyperkube kubelet \
 	--healthz-port=10248 \
 	--cert-dir=/var/lib/p8s-kubelet/pki \
 	--kubeconfig={{.RootABSPath}}/manifest-config/kubeconfig-insecure.yaml \
-	--cloud-provider="" \
```

Comment:
Are we planning on supporting this flag at some point ? (referring to @kindermoumoute's issue)

Reply:
Yes we should, but it's not something needed for our use case right now.